### DISCUSSION TITLE
Comment before the first move - not showing up

### DISCUSSION THREAD

**Discussion by sbernsto** [mliebelt/pgn-viewer#550] [General]
I was curious if anyone else has come across this.  I'm seeing that If I have a comment prior to the first move being made, PGN-viewer does not show that comment.  

Example: 
Example PGN:
[Event "London Opening: Introduction"]
[Site "https://lichess.org/study/pLBSiPGF/wdGlcwDu"]
[Result "*"]
[Variant "Standard"]
[ECO "D00"]
[Opening "Queen's Pawn Game: Accelerated London System"]
[Annotator "https://lichess.org/@/Sbernsto"]
[UTCDate "2024.05.20"]
[UTCTime "15:09:38"]

{ Hi! You've probably heard about an opening for white called the London System. In this study, I will teach you the basics of this popular opening. } 1. d4 { We will be starting with d4. It is not as aggressive as e4 can be but d4 is a good middleground between aggressive and solid games.

The result that shows only the comments after 1.d4 and nothing before
![image](https://github.com/mliebelt/pgn-viewer/assets/6393618/c2270f56-1b14-4294-a700-3cb2a9241fd2)

**Comment by mliebelt** [comment]
If that is the case, it qualifies as a bug. I have added support for exactly one comment before game starts, because that is a common case. I will try to create a unit test from your input, that first checks that the parser is handling it correctly, and then a visual example (named `ticket550.html`) to show it. Will take some time, because I am busy lately ...

Thank you for sharing!

**Comment by mliebelt** [comment]
See ticket #555

**Comment by mliebelt** [comment]
Closed that discussion, I think that issue is fixed now.
